zoo-css-challenge/index.html

```diff
@@ -51,16 +54,16 @@ <h2>Bears</h2>
 		</section>
 
 		<section class="tigers">
-			<h2>Tiger</h2>
-			<h3>Top 5 Tiger Facts</h3>
-			<ol>
+			<h2>Tiger Facts</h2>  <!-- Change the heading-->
+			<!-- <h3>Top 5 Tiger Facts</h3> -->
```

Comment:
```suggestion
```
Software teams usually don't keep commented code. We should remove commented code